### PR TITLE
[bitnami/harbor] Adds different credential fields for each database

### PR DESCRIPTION
**Pull request by schnatterer**
**Description of the change**

This PR adds the option of setting different credentials for each database. If no value is provided the field `externalDatabase.user` and/or `externalDatabase.password` is used. So this does not break compatibility.

**Benefits**

In case of a breach only one database is compromised.

**Possible drawbacks**

**Applicable issues**

**Additional information**

Some time ago, we've create a PR for the official harbor helm chart (without success), See goharbor/harbor-helm#501 and have since changed to bitnami/harbor.

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by andresbono** [comment]
Hi @schnatterer, thanks for your contribution! :tada:

We will review this PR shortly.

File: bitnami/harbor/templates/_helpers.tpl [hunk 1]
```diff
@@ -101,6 +101,42 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
   {{- end -}}
 {{- end -}}
 
+{{- define "harbor.database.clairUsername" -}}
+  {{- if eq .Values.postgresql.enabled true -}}
+    {{- printf "%s" "postgres" -}}
```
Comment (andresbono) [inline bitnami/harbor/templates/_helpers.tpl:106]: Shouldn't this be: 
```suggestion
    {{- .Values.postgresql.postgresqlUsername -}}
```

The same for the rest if you think this change is appropriate.

Comment (schnatterer) [reply bitnami/harbor/templates/_helpers.tpl]: You're right. I missed that one, when porting this PR from the official helm chart.
The same applies to `notaryServerUsername` and `notarySignerUsername`.

**Comment by schnatterer** [comment]
@andresbono thanks for your review.
Rebased and implemented your suggestion.

**Pull request merged**